.gitignore

```diff
@@ -23,4 +22,5 @@ yarn-error.log
 !.yarn/plugins
 !.yarn/releases
 !.yarn/sdks
-!.yarn/versions
\ No newline at end of file
+!.yarn/versions
+/wire-webapp.iml
```

Comment:
please add those kind of files to your personal global `.gitignore`. They don't belong in the repo directly